src/lib/functions.ts

```diff
@@ -26,6 +26,7 @@ const OutboundMessageSchema = {
     },
     channel: { type: 'string' },
     server: { type: 'string' },
+    query: {}
   }
```

Comment:
Let's validate `query` here too. We now know its type.

Reply:
I have created query as `object` type . And since we dont know what the user is going to pass in the query param i cant define the type of keys and values here. I have added the implementation according to my understanding please have a look.